provisioning/miniprov/hare_mp/main.py

```diff
@@ -53,12 +59,32 @@ def execute(cmd: List[str]) -> str:
     return out
 
 
-def setup_logging():
-    logging.basicConfig(level=logging.INFO, format='%(message)s')
+def create_logger_directory():
+    """Create log directory if not exsists."""
+    _logger_directory = os.path.join(haredeploy_log_directory)
+    if not os.path.isdir(_logger_directory):
+        try:
+            os.mkdir(_logger_directory)
+        except BaseException:
+            raise Exception(f"{_logger_directory} Could not be created")
```

Comment:
The user will not see the actual reason why the directory could not be created. That information could have been found in the caught exception but it is ignored.

Reply:
Correct, please look at `logging.exception`.

Reply:
done